src/nupic/regions/VectorFile.cpp

```diff
@@ -29,7 +29,6 @@
 #include <cstdio> //fopen
 #include <iostream>
 #include <math.h>
-#include <nupic/math/Utils.hpp> // For isSystemLittleEndian and utils::swapBytesInPlace.
```

Comment:
these were the only uses of math/Utils,
and I could remove the support (for reading files with different endian-ness, as we can only serialize to/from files on the same platform now)